src/component/Header/index.jsx

```diff
@@ -1,9 +1,27 @@
-import React from 'react';
+import Nav from 'react-bootstrap/Nav'
+import Navbar from 'react-bootstrap/Navbar'
+import Image from 'react-bootstrap/Image'
+import styles from './Header.module.css'
+import PomoLogo from '../../Logo/PomoLogo.png'
+import SettingLogo from '../../Logo/SettingLogo.png'
+import LoginLogo from '../../Logo/LoginLogo.png'
```

Comment:
These are images and not classes, so they should use camelCase. 
```suggestion
import pomoLogo from '../../Logo/PomoLogo.png'
import settingLogo from '../../Logo/SettingLogo.png'
import loginLogo from '../../Logo/LoginLogo.png'
```